src/main/kotlin/no/nav/medlemskap/regler/v2/arbeidsforhold/RegelgruppeForArbeidsforhold.kt

```diff
@@ -0,0 +1,147 @@
+package no.nav.medlemskap.regler.v2.arbeidsforhold
+
+import no.nav.medlemskap.domene.Arbeidsforholdstype
+import no.nav.medlemskap.domene.Skipsregister
+import no.nav.medlemskap.regler.common.Funksjoner.erDelAv
+import no.nav.medlemskap.regler.common.Funksjoner.erIkkeTom
+import no.nav.medlemskap.regler.common.Personfakta
+import no.nav.medlemskap.regler.common.Resultattype.*
+import no.nav.medlemskap.regler.v2.common.*
+
+class RegelgruppeForArbeidsforhold(val personfakta: Personfakta) : Regelgruppe() {
+
+    override fun evaluer() =
+            Spørsmål(
+                    identifikator = "1.5",
+                    spørsmål = "Oppfyller arbeidsforholdet kravet?",
+                    beskrivelse = "",
+                    operasjon = { kjørRegler() }
+            ).utfør()
+
+
+    private fun kjørRegler(): Svar {
+        val arbeidsforhold = harArbeidsforhold.utfør()
+
+        if (arbeidsforhold.svar.resultat == NEI) {
+            return Svar(
+                    resultat = UAVKLART,
+                    begrunnelse = "Brukeren mangler registrerte arbeidsforhold",
+                    underspørsmål = listOf(arbeidsforhold)
+            )
+        }
+
+        val norskArbeidsgiver = erArbeidsgiverNorsk.utfør()
+
+        if (norskArbeidsgiver.svar.resultat == NEI) {
+            return Svar(
+                    resultat = UAVKLART,
```

Comment:
Nei, det var åpenbart ikke sånn.. Hm.. Dette var ikke helt intuitivt for meg. Jeg tror jeg må dra ned koden og se på dette i IntelliJ.